tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -26,6 +26,13 @@ Then('the error message {string} should be displayed on the webUI', function (fo
     .expect.element('@message').text.to.equal(folder)
 })
 
+Then('the success message {string} should be displayed on the webUI', function (message) {
```

Comment:
```suggestion
Then('the success/error message {string} should be displayed on the webUI', function (message) {
```
And remove the similar method above this.